packages/framework/src/Model/Customer/User/CustomerUserFacade.php

```diff
@@ -486,4 +486,13 @@ protected function createCustomerWithAddresses(CustomerUserUpdateData $customerU
 
         return $customer;
     }
+
+    /**
+     * @param int $salesRepresentativeId
+     * @return array
```

Comment:
this returns an array of emails, right? So I would use here `@return string[]` and consider naming the method accordingly

Reply:
done, opted for findEmailsOfCustomerUsersUsingSalesRepresentative, hope it is ok